### DISCUSSION TITLE
Unable to load rtx.asi Error: 126

### DISCUSSION THREAD

**Discussion by Lrtzeh** [xoxor4d/p2-rtx#24] [General]
I have no idea where I should ask for support on github as I've practically never used it, sorry if this is in the wrong spot for posting this.

![image](https://github.com/user-attachments/assets/b5b1f085-e320-4825-a569-5d71276ccd0e)

I've tried googling this error to no avail. I have no idea how to use "Dependencies".

I am an extreme noob to RTX Remix and github, so I might just be missing something super obvious. I am running it through _start_portal2_rtx.bat. 

After clicking "Ok", the game launches and shows the RTX Remix "hit alt+x" menu and can open up, but I don't think it's actually on. It looks almost entirely vanilla. The window title says PORTAL 2 - Direct3D 9. I have changed my graphics settings to the settings recommended. I'm fairly certain I installed this correctly, dragging out the contents within both of the portal2-root folders into the portal2 directory. I've uninstalled and reinstalled Portal 2 and I still get this issue. 

Portal with RTX does work for me without any issues. I'm most likely a dummy and did not do something correctly.

Windows 11 Home 23H2
AMD Ryzen 9 7900X
32GB DDR5 6000
Geforce RTX 3070

**Comment by Justsnoopy30** [comment]
See https://github.com/xoxor4d/p2-rtx/issues/6#issuecomment-2489705279 to fix this
> Download [this](https://www.microsoft.com/en-ie/download/details.aspx?id=8109) and run the exe, after that open DXSETUP.EXE in the directory you selected and it should install some stuff which you need.

**Comment by Lrtzeh** [comment]
> See [#6 (comment)](https://github.com/xoxor4d/p2-rtx/issues/6#issuecomment-2489705279) to fix this
> 
> > Download [this](https://www.microsoft.com/en-ie/download/details.aspx?id=8109) and run the exe, after that open DXSETUP.EXE in the directory you selected and it should install some stuff which you need.

Thank you, this worked. Sorry for being blind, I have practically no experience on github.

I have a new question though, it runs a lot worse than Portal with RTX for me, to the point it's practically unplayable unless I set DLSS to ultra performance (even on 720p) and it looks like a oily ai generated mess. Around 15-25fps on 1280x720 on DLSS Quality and 5-15 fps on 2560x1080 DLSS Quality. Using the other upscalers does not help either. Is there some magic setting i'm missing or is this to be expected with my specs? If it is to be expected, is there a config file or something that would help FPS a bit? I would love to send logs but I have no idea how to do so.

Also, for some reason, when switching back and forth between different DLSS presets (ultra performance, performace, quality, etc.) the VRAM usage went to 100% and stayed there even when switching back to ultra performance. Previously, it was only at around ~63%. Nothing except restarting the game fixed it. Idk if this is just an RTX Remix bug or with P2RTX. Not a huge deal since rebooting the game takes literal seconds lol.

**Comment by xoxor4d** [comment]
Portal 2 maps are much bigger and more detailed then the maps of Portal 1 and the amount of culling was drastically reduced to get rid of as much light culling as possible. Multi threaded rendering had to be disabled as well.

The latest github action build should slightly improve performance and I'll try to find ways to further improve the performance.

> VRAM usage went to 100% and stayed there even when switching back to ultra performance
Sounds like a remix issue

If you have _questions_, please use the `Discussions` tab (converted this to a discussion)

**Comment by Lrtzeh** [comment]
Thank you for letting me know and for the help.
Sorry, what is the latest github action build? Where do I find this?
